.github/workflows/release.yml

```diff
@@ -6,6 +6,9 @@ on:
         description: "The tag of release"
         required: true
 
+env:
```

Comment:
Does this need to be available on the whole workflow? I'd prefer to scope it to just the brew command so something like git checkout couldn't exfiltrate it. Also, what permissions are needed? I ask because the workflow token has some write permissions we might not want to make available

Reply:
I'll move this down. I also swapped from the workflow token to just `GITHUB_TOKEN` which I think has less scope?